baseband/dada/base.py

```diff
@@ -430,19 +430,31 @@ def open(name, mode='rs', subset=None, header=None, **kwargs):
     is_template = isinstance(name, six.string_types) and ('{' in name and
                                                           '}' in name)
     is_sequence = isinstance(name, (tuple, list))
+    is_sequentialfile = isinstance(name, (sf.SequentialFileReader,
+                                          sf.SequentialFileWriter))
 
     if 'b' not in mode:
         if header is None:
             if 'w' in mode:
+                # If sequentialfile object, check that it's opened for writing.
+                if is_sequentialfile:
+                    assert name.mode == 'w+b', (
+                        'open only accepts sequential files opened in '
+                        '\'w+b\' mode for writing.')
                 # For writing a header is required.
-                if 'nthread' in kwargs:
```

Comment:
Just a left-over that we forgot to remove earlier, correct?

Reply:
Yes, removing to get rid of the notion that the first axis of any sample shape is a thread.  That property is still available if the user passes `npol`.